Makefile

```
@@ -1578,14 +1578,15 @@ endif
 .PHONY: protos-up-to-date/host
 protos-up-to-date/host: must-start-clean/host grpc/host
 	@if ! git diff --quiet; then \
-		./build.assets/tooling/cmd/please-run.sh "protos gRPC" "make grpc"; \
+		./build.assets/please-run.sh "protos gRPC" "make grpc"; \
 		exit 1; \
 	fi
 
 .PHONY: must-start-clean/host
 must-start-clean/host:
 	@if ! git diff --quiet; then \
-		./build.assets/tooling/cmd/please-run.sh "protos gRPC" "make grpc"; \
+		echo 'This must be run from a repo with no unstaged commits.'; \
```

```suggestion
		@echo 'This must be run from a repo with no unstaged commits.'; \
```

otherwise it echoes the echo first.